state-management/state-manage-assign/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+
+import React from 'react';
+import './App.css';
+
+
+import BasicTable  from './components/Table';
+
+
+function App() {
+
+
+
+  return (
+    <div className='App'>
+      <div className='container'>
+        
+        <BasicTable ></BasicTable>
+      </div>
+    </div>
+    
+    
+  );
+}
+
+export default App;
```

Comment:
For now I am approving but make sure no extra space will be there in code.
anywhere.

Reply:
okay